lib/core/db_models/old/job_summary_data_model.dart

```diff
@@ -25,44 +25,46 @@ class JobSummaryDataModel {
     @required Map<String, dynamic> map,
     @required String docID,
   }) {
+    print("data map is ${map}");
+    print("data doc id is ${docID}");
     if (map.isNotEmpty) {
       try {
         String _organisationID = '';
         final DocumentReference _organisationRef =
             map[JobSummarySchema.organisationRef];
+        print("organisation ref ${_organisationRef ?? null}");
         if (_organisationRef != null) {
           _organisationID = _organisationRef.id;
           String _jobID = '';
           final DocumentReference _jobRef = map[JobSummarySchema.jobRef];
-          if (_jobRef != null) {
-            _jobID = _jobRef.id;
+          print("Job ref ${_jobRef ?? null}");
 
-            final JobDataModel _jobDataModel = JobDataModel.fromMap(
-                map: map[JobSummarySchema.jobData] ?? {}, docID: _jobID
-            );
-            final Map<String, dynamic> workers =
-                map[JobSummarySchema.workers] ?? {};
-            List<WorkerModel> _allWorkerArray = [];
-            workers.forEach((key, element) {
-              final WorkerModel _worker = WorkerModel.fromMap(
-                map: element,
-                docID: key,
-              );
+          _jobID = docID;
```

Comment:
docID should be renamed as docId. Also if _jobID and docID is same, please remove _jobID from model